### PR TITLE
Enable R2R in C for all CI except Windows

### PR DESCRIPTION
**Pull request by thestr4ng3r**

### CODE REVIEW

**Comment by XVilka** [comment]
See related https://github.com/radareorg/radare2/issues/16285

**Comment by radare** [comment]
calling to all the windows devs to make the new r2r.c work on windows before friday!!!11

**Comment by thestr4ng3r** [comment]
If there is no impl for windows on friday, windows support of radare2 and cutter will be dropped.

**Comment by radare** [comment]
I would say to drop r2r.v right now, well just move it into a separate repo and focus in r2r.js and .c, then kill the new and src directories

**Comment by XVilka** [comment]
> focus in r2r.js

Focus on removing r2r.js you meant?

**Comment by XVilka** [comment]
Failed on OSX:
```
[XX] db/tools/rabin2 rabin2 -D swift
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc '!rabin2 -D swift  _TFSSCfT21_builtinStringLiteralBp8byteSizeBw7isASCIIBi1__SS' ../bins/elf/libc.so.0
-- stdout
--- .a	2020-03-31 10:47:43.000000000 +0000
+++ .b	2020-03-31 10:47:43.000000000 +0000
@@ -1 +1 @@
-Swift.String.init (_builtinStringLiteral(Builtin.RawPointer byteSize__Builtin.Word isASCII__Builtin.Int1 _) -> String
+Swift.String.init(_builtinStringLiteral: Builtin.RawPointer, byteSize: Builtin.Word, isASCII: Builtin.Int1) -> Swift.String


[XX] db/cmd/cmd_i iD
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'iD swift _TFSSCfT21_builtinStringLiteralBp8byteSizeBw7isASCIIBi1__SS' ../bins/elf/libmagic.so
-- stdout
--- .a	2020-03-31 10:48:00.000000000 +0000
+++ .b	2020-03-31 10:48:00.000000000 +0000
@@ -1 +1 @@
-Swift.String.init (_builtinStringLiteral(Builtin.RawPointer byteSize__Builtin.Word isASCII__Builtin.Int1 _) -> String
+Swift.String.init(_builtinStringLiteral: Builtin.RawPointer, byteSize: Builtin.Word, isASCII: Builtin.Int1) -> Swift.String
```

**Comment by XVilka** [comment]
Failed on OpenBSD:
```
[XX] db/tools/rasm2 rasm2 #1167 3
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc '!rasm2 -s att -a x86.as -b 64 "test %rbx, %rax" ' -
-- stdout
--- .a	Tue Mar 31 10:45:30 2020
+++ .b	Tue Mar 31 10:45:30 2020
@@ -1 +1 @@
-4885d8
+4885c3


[XX] db/tools/rasm2 rasm2 #1167 4
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc '!rasm2 -s intel -a x86.as -b 64 "test rax, rbx" ' -
-- stdout
--- .a	Tue Mar 31 10:45:30 2020
+++ .b	Tue Mar 31 10:45:30 2020
@@ -1 +1 @@
-4885d8
+4885c3


[XX] db/cmd/cmd_print prc text color
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc '. ../bins/other/palette.r2
.. ../bins/src/prc_256.py
e scr.color=3
prc 256
' -
-- stdout
--- .a	Tue Mar 31 10:54:58 2020
+++ .b	Tue Mar 31 10:54:58 2020
@@ -1,16 +1,16 @@
-0x00000000 000102030405060708090a0b0c0d0e0f
-0x00000010 101112131415161718191a1b1c1d1e1f
-0x00000020 202122232425262728292a2b2c2d2e2f
-0x00000030 303132333435363738393a3b3c3d3e3f
-0x00000040 404142434445464748494a4b4c4d4e4f
-0x00000050 505152535455565758595a5b5c5d5e5f
-0x00000060 606162636465666768696a6b6c6d6e6f
-0x00000070 707172737475767778797a7b7c7d7e7f
-0x00000080 808182838485868788898a8b8c8d8e8f
-0x00000090 909192939495969798999a9b9c9d9e9f
-0x000000a0 a0a1a2a3a4a5a6a7a8a9aaabacadaeaf
-0x000000b0 b0b1b2b3b4b5b6b7b8b9babbbcbdbebf
-0x000000c0 c0c1c2c3c4c5c6c7c8c9cacbcccdcecf
-0x000000d0 d0d1d2d3d4d5d6d7d8d9dadbdcdddedf
-0x000000e0 e0e1e2e3e4e5e6e7e8e9eaebecedeeef
-0x000000f0 f0f1f2f3f4f5f6f7f8f9fafbfcfdfeff
+0x00000000                                 
+0x00000010                                 
+0x00000020                                 
+0x00000030                                 
+0x00000040                                 
+0x00000050                                 
+0x00000060                                 
+0x00000070                                 
+0x00000080                                 
+0x00000090                                 
+0x000000a0                                 
+0x000000b0                                 
+0x000000c0                                 
+0x000000d0                                 
+0x000000e0                                 
+0x000000f0                                 


-- stderr
sh: python: not found
Cannot open ttyname(0) (null)


Finished in 19 minutes and 27 seconds.
gmake: *** [Makefile:30: r2r-tests] Error 1
```

**Comment by XVilka** [comment]
Please also update the [`test/README.md`](https://github.com/radareorg/radare2/blob/master/test/README.md)

File: .builds/freebsd.yml [hunk 1]
```diff
@@ -5,22 +5,33 @@ packages:
     - devel/gmake
     - devel/llvm
     - ftp/wget
+    - python
```
Comment (XVilka) [inline .builds/freebsd.yml:8]: I think it's `lang/python` instead: https://svnweb.freebsd.org/ports/head/lang/python/

**Pull request merged**